.github/workflows/mypy.yaml

```diff
@@ -1,6 +1,6 @@
 name: Run mypy
 
-on:
+"on":
```

Comment:
We fix this in daf_butler by removing the truthiness test for github actions: https://github.com/lsst/daf_butler/blob/main/.yamllint.yaml#L14-L17